backend/fetcher.go

```diff
@@ -38,15 +38,28 @@ func newFetcher(cs ConfigStore, ts TemplateStore, r ParamsResolver) *fetcher {
 	return &fetcher{cs, ts, r}
 }
 
+func (f *fetcher) FetchAll() ([]client.Stack, error) {
+	stackConfigs, err := f.cs.FetchAll()
+	if err != nil {
+		return []client.Stack{}, fmt.Errorf("unable to fetch stacks: %s", err)
+	}
+
+	return f.fetchTemplates(stackConfigs)
+}
+
 func (f *fetcher) Fetch(name string) ([]client.Stack, error) {
```

Comment:
Should we maybe just update these to return a `client.StackList`?

Reply:
Yes probably. Didn’t know if it was a CLI concern or not but doesn’t hurt.

Reply:
Hmmm, yeah, that's a good point. I think for now it makes sense, if only for the convenience.

Really though, I think we probably should be returning concrete types here.